src/misc/news/items/UsageOptInNews.ts

```diff
@@ -0,0 +1,75 @@
+import {NewsListItem} from "../NewsListItem.js"
+import m, {Children} from "mithril"
+import {NewsId} from "../../../api/entities/tutanota/TypeRefs.js"
+import {locator} from "../../../api/main/MainLocator.js"
+import {InfoLink, lang} from "../../LanguageViewModel.js"
+import {logins} from "../../../api/main/LoginController.js"
+import {Dialog} from "../../../gui/base/Dialog.js"
+import {Button, ButtonAttrs, ButtonType} from "../../../gui/base/Button.js"
+
+export default class UsageOptInNews extends NewsListItem {
+	isShown(): boolean {
+		return locator.usageTestModel.showOptInIndicator()
+	}
+
+	render(newsId: NewsId): Children {
+		const lnk = InfoLink.Privacy
+		const userSettingsGroupRoot = logins.getUserController().userSettingsGroupRoot
+
+		const closeAction = (optedIn?: boolean) => {
+			this.acknowledge(newsId).then(() => {
+				if (optedIn) {
+					Dialog.message("userUsageDataOptInThankYouOptedIn_msg")
+				} else if (optedIn !== undefined) {
+					Dialog.message("userUsageDataOptInThankYouOptedOut_msg")
+				}
+			})
+		}
+
+		const buttonAttrs: Array<ButtonAttrs> = [
+			{
+				label: "decideLater_action",
+				click: () => closeAction(),
+				type: ButtonType.Secondary,
+			},
+			{
+				label: "deactivate_action",
+				click: () => {
+					userSettingsGroupRoot.usageDataOptedIn = false
```

Comment:
Shouldn't this be a part of `usageTestModel` somehow? or maybe `userController`?